packages/wiz-ui-react/src/components/base/popup/components/popup.tsx

```diff
@@ -41,6 +41,17 @@ type Props = {
   children: ReactNode;
 } & ComponentProps<"div">;
 
+/** 与えられた要素が、fixedまたはstickyの要素上にあるかどうかを返します。 */
+const hasFixedOrStickyParent = (el: HTMLElement | null): boolean => {
+  const checkParent = (el: HTMLElement | null): HTMLElement | null => {
+    if (!el) return null;
+    const position = window.getComputedStyle(el).position;
+    if (position === "fixed" || position === "sticky") return el;
+    return checkParent(el.parentElement);
+  };
+  return !!checkParent(el);
+};
```

Comment:
boolean を返す形で記述すれば内部の関数を定義する必要は特に無くなるように思いました。

```suggestion
const hasFixedOrStickyParent = (el: HTMLElement | null): boolean => {
  if (!el) {
    return false;
  }
  const position = window.getComputedStyle(el).position;
  if (position === "fixed" || position === "sticky") {
    return true;
  }
  return hasFixedOrStickyParent(el.parentElement);
};
```

Reply:
[58d4081](https://github.com/Wizleap-Inc/wiz-ui/pull/980/commits/58d4081b59be327721aaa4a6051d709fdc5277c0) にて対応いたしました！
`!!`で無理やりbooleanに変換する必要がなくなってスッキリしました🙏